lib/system_description.rb

```diff
@@ -276,19 +276,28 @@ def runs_service?(name)
   end
 
   def has_file?(name)
-    self["config_files"].files.any? { |file| file.name == name } ||
-      self["unmanaged_files"].files.any? { |file| file.name == name }
+    EXTRACTABLE_SCOPES.each do |scope|
+      found = self[scope].has_file?(name)
+      return true if found
```

Comment:
Same here. `return true if self[scope].has_file?(name)` would do the trick as well.